packages/preview/shane-hhu-thesis/0.1.0/init-files/thesis.typ

```diff
@@ -0,0 +1,316 @@
+#import "@preview/shane-hhu-thesis:0.1.0": degree-conf, thanks, appendix
```

Comment:
If I'm not mistaken the first item here should be  `bachelor-conf`.

Reply:
Sorry. I fixed it.